src/SaveHandler/MongoDB.php

```diff
@@ -57,8 +57,10 @@ public function __construct($mongo, MongoDBOptions $options)
     {
         if (!($mongo instanceof \MongoClient || $mongo instanceof \Mongo)) {
             throw new InvalidArgumentException(
-                'Parameter of type %s is invalid; must be MongoClient or Mongo',
-                (is_object($mongo) ? get_class($mongo) : gettype($mongo))
+                sprintf(
+                    'Parameter of type %s is invalid; must be MongoClient or Mongo',
+                    (is_object($mongo) ? get_class($mongo) : gettype($mongo))
+                )
```

Comment:
> Where is this documented?

Good question for a framework where IMHO documentation is a problem :smile:

Looking at other ZF2 code in almost every situation the indentation I mentioned is used: https://github.com/zendframework/zend-session/search?utf8=%E2%9C%93&q=Exception%28sprintf%28